app/main.py

```diff
@@ -1,5 +1,71 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
```

Comment:
Sorry, I didn't say you before. But coords could be both types, for that purpose use annotation like this. The same in other similar cases

```suggestion
            coords: list | None = None
```